### PR TITLE
CosmosDB management SDK 2022-02-15-preview

### PR DESCRIPTION
**Pull request by amar-sagare**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by amar-sagare** [comment]
@archerzz  can you please review this PR?

**Comment by amar-sagare** [comment]
waiting for firhter changes please do not merge.

**Comment by amar-sagare** [comment]
@archerzz  can you please review this. Thanks!

**Review by archerzz — CHANGES_REQUESTED** [review]

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/RestorableDatabaseAccountGetResult.cs [hunk 1]
```diff
@@ -53,10 +55,11 @@ public RestorableDatabaseAccountGetResult()
         /// <param name="type">The type of Azure resource.</param>
         /// <param name="location">The location of the resource group to which
         /// the resource belongs.</param>
-        public RestorableDatabaseAccountGetResult(string accountName = default(string), System.DateTime? creationTime = default(System.DateTime?), System.DateTime? deletionTime = default(System.DateTime?), string apiType = default(string), IList<RestorableLocationResource> restorableLocations = default(IList<RestorableLocationResource>), string id = default(string), string name = default(string), string type = default(string), string location = default(string))
+        public RestorableDatabaseAccountGetResult(string accountName = default(string), System.DateTime? creationTime = default(System.DateTime?), System.DateTime? oldestRestorableTime = default(System.DateTime?), System.DateTime? deletionTime = default(System.DateTime?), string apiType = default(string), IList<RestorableLocationResource> restorableLocations = default(IList<RestorableLocationResource>), string id = default(string), string name = default(string), string type = default(string), string location = default(string))
```
Comment (archerzz) [inline sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/RestorableDatabaseAccountGetResult.cs:58]: breaking change: new parameters inserted in existing parameters

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/RestorableSqlContainerPropertiesResourceContainer.cs [hunk 2]
```diff
@@ -53,14 +55,15 @@ public RestorableSqlContainerPropertiesResourceContainer()
...
-        public RestorableSqlContainerPropertiesResourceContainer(string id, IndexingPolicy indexingPolicy = default(IndexingPolicy), ContainerPartitionKey partitionKey = default(ContainerPartitionKey), int? defaultTtl = default(int?), UniqueKeyPolicy uniqueKeyPolicy = default(UniqueKeyPolicy), ConflictResolutionPolicy conflictResolutionPolicy = default(ConflictResolutionPolicy), long? analyticalStorageTtl = default(long?), string _rid = default(string), double? _ts = default(double?), string _etag = default(string), string _self = default(string))
+        public RestorableSqlContainerPropertiesResourceContainer(string id, IndexingPolicy indexingPolicy = default(IndexingPolicy), ContainerPartitionKey partitionKey = default(ContainerPartitionKey), int? defaultTtl = default(int?), UniqueKeyPolicy uniqueKeyPolicy = default(UniqueKeyPolicy), ConflictResolutionPolicy conflictResolutionPolicy = default(ConflictResolutionPolicy), ClientEncryptionPolicy clientEncryptionPolicy = default(ClientEncryptionPolicy), long? analyticalStorageTtl = default(long?), string _rid = default(string), double? _ts = default(double?), string _etag = default(string), string _self = default(string))
```
Comment (archerzz) [inline sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/RestorableSqlContainerPropertiesResourceContainer.cs:58]: same breaking change

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/SqlContainerResource.cs [hunk 3]
```diff
@@ -42,15 +42,18 @@ public SqlContainerResource()
...
         /// <param name="conflictResolutionPolicy">The conflict resolution
         /// policy for the container.</param>
+        /// <param name="clientEncryptionPolicy">The client encryption policy
+        /// for the container.</param>
         /// <param name="analyticalStorageTtl">Analytical TTL.</param>
-        public SqlContainerResource(string id, IndexingPolicy indexingPolicy = default(IndexingPolicy), ContainerPartitionKey partitionKey = default(ContainerPartitionKey), int? defaultTtl = default(int?), UniqueKeyPolicy uniqueKeyPolicy = default(UniqueKeyPolicy), ConflictResolutionPolicy conflictResolutionPolicy = default(ConflictResolutionPolicy), long? analyticalStorageTtl = default(long?))
+        public SqlContainerResource(string id, IndexingPolicy indexingPolicy = default(IndexingPolicy), ContainerPartitionKey partitionKey = default(ContainerPartitionKey), int? defaultTtl = default(int?), UniqueKeyPolicy uniqueKeyPolicy = default(UniqueKeyPolicy), ConflictResolutionPolicy conflictResolutionPolicy = default(ConflictResolutionPolicy), ClientEncryptionPolicy clientEncryptionPolicy = default(ClientEncryptionPolicy), long? analyticalStorageTtl = default(long?))
```
Comment (archerzz) [inline sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/SqlContainerResource.cs:48]: same breaking change

**Review by archerzz — APPROVED** [review]
approved, given the SDK is still in `preview` stage.

**Comment by chidozieononiwu** [comment]
Looks like this was merged into the wrong directory. Tis needs to be reverted and added into https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB

@weshaggard

**Comment by ArthurMa1978** [comment]
> Looks like this was merged into the wrong directory. Tis needs to be reverted and added into https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB
> 
> @weshaggard

should be fixed

**Pull request merged**